### PR TITLE
Fix arrow for Select

### LINKED ISSUE #371: Missing arrow at Select component
I'm updating my project from rebass 1.x and I note arrow is not present into select

screen shot 2018-03-07 at 22 43 00

I suppose the inline svg as background-image is buggy

### PR DESCRIPTION
**Pull request by Kikobeats**
Closes #371 

Inspired by: https://codepen.io/herrfischer/pen/aNpWwy

### CODE REVIEW

**Comment by Kikobeats** [comment]
ping to @jxnblk !

File: src/Select.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import sys from 'system-components'
 import { themeGet } from 'styled-system'
 
-const arrow = encodeURI(`data:image/svg+xml,<svg xmlns='http://www.w3c.org/2000/svg' width='32' height='32' viewbox='0 0 32 32' fill='tomato'> <path d='M0 6 L32 6 L16 28 z' /> </svg>`)
+const arrow = encodeURIComponent(`<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100'><polygon points='0,0 100,0 50,50'/></svg>`)
```
Comment (jxnblk) [inline src/Select.js]: I'd rather keep the original SVG here, since that doesn't seem to be the issue this PR is intended to solve

Comment (Kikobeats) [reply src/Select.js]: done, feel free to edit the PR 🙂

**Comment by jxnblk** [comment]
Thanks! I think snapshots need to be updated with `npm t -- -u`

**Comment by Kikobeats** [comment]
@jxnblk done! sorry for the delay

What do you need for releasing v2.x officially? 🙂

**Comment by jxnblk** [comment]
Awesome thanks! For v2, just a couple small issues and mostly docs for the official release

**Pull request merged**